regression/contracts-dfcc/missing-function-body/main.c

```diff
@@ -0,0 +1,13 @@
+int foo(int);
+
+int bar(int i) __CPROVER_requires(i > 0)
+  __CPROVER_ensures(__CPROVER_return_value > 0)
+{
+  return foo(i);
+}
+
+int main()
+{
+  int i;
+  bar(i);
+}
```

Comment:
Nit pick: GitHub says there is no newline here.

Reply:
fixed